UnitsNet/UnitsNet.Net35.csproj

```diff
@@ -40,7 +40,158 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs;" Exclude="obj\**;**\*.WindowsRuntimeComponent.cs" />
+    <Compile Include="AmbiguousUnitParseException.cs" />
```

Comment:
uh this might have been my doing. I'm not sure. I had some problems with my visual studio reloading the auto generated files. I had added them manually but then discovered that you used the wildcard in the project file script. I then reset the script back to it's original